app/decorators/decidim/user_decorator.rb

```diff
@@ -12,11 +12,28 @@
                           association_foreign_key: :decidim_area_id,
                           validate: false
 
+  has_and_belongs_to_many :participatory_processes,
+                          join_table: :decidim_participatory_process_user_roles,
+                          foreign_key: :decidim_user_id,
+                          association_foreign_key: :decidim_participatory_process_id,
+                          validate: false
+
+  has_and_belongs_to_many :assemblies,
+                          join_table: :decidim_assembly_user_roles,
+                          foreign_key: :decidim_user_id,
+                          association_foreign_key: :decidim_assembly_id,
+                          validate: false
+
   scope :admins, -> { where(admin: true) }
   scope :user_managers, -> { where(roles: ["user_manager"]) }
   scope :department_admins, -> { where(roles: ["department_admin"]) }
+  scope :space_administrator, -> { where(roles: ["department_admin"]) }
 
   def department_admin?
     role?('department_admin')
   end
+
+  def space_administrator?
+    role?('space_administrator')
```

Comment:
where is that this role is assigned to users?
also `space_administrator` name is different from the one used in other places like:
- app/queries/decidim/admin/user_admin_filter.rbL61
- app/views/decidim/admin/users/_form.html.erb, 
- config/initializers/user_roles.rbL2